### ISSUE TITLE
Github Student Pack - Namecheap custom domain

### ISSUE THREAD

**Issue by zxstai** [Slotshaven-programmering2019/Matador2019#17]
Jeg tænkte om det var muligt at vi kunne med brug af vores GitHub student pack, altså namecheaps aftale med at der er 1 års gratis domain.

Det skal blive forstået som at vi kan tilgå fx, Slotshaven-Matador.me (.me tror jeg er det eneste gratis top level domain de giver ud, men det kan jeg lige følge op på)

Det er bare en lille detalje, men jeg synes den er relevant for brugen af IT værktøjer som vi har tilgængelig.

**Comment by laererlund** [comment]
Jeg er ikke helt med på hvad det er der er tanken - ville du anskaffe et domæne specifikt til dette spil?

**Comment by zxstai** [comment]
Ja, med GitHub Student pack har Namecheap et gratis domæne som kan tilknyttes ens Github projekt

Så man tilgår fx: www.slotshavenmatador.me/
det domæne er så linked til vores matador spil
